addons/binding/org.openhab.binding.omnikinverter/src/main/java/org/openhab/binding/omnikinverter/internal/OmnikInverterHandlerFactory.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
```

Comment:
Don't want to be picky on this one, but openhab is migrating to EPLv2 license. CC @kaikreuzer

Reply:
hmm okay,would like to get some definite recommendation on this one. This is what the initial binding template gave me.

Reply:
Will follow @wborn's pointers on this issue thanks.